[Auto] DOCS-14720: Cloud Security Vulnerabilities - Document saved views workaround on Findings page (#37770)
DataDog/documentation: content/en/security/cloud_security_management/vulnerabilities/_index.md

For vulnerabilities with available fixes, the Findings page provides guided remediation steps to assist Dev and Ops teams in resolving issues more quickly and effectively. You can also triage, mute, comment, and assign vulnerabilities to manage their lifecycle.

> <div class="alert alert-info">The Vulnerabilities Findings page does not include a <strong>Save as new view</strong> option. To reuse a filter configuration, bookmark the full page URL — your search query and facet selections are preserved in the URL.</div>

Reviewer: ```suggestion
<div class="alert alert-info">To reuse your explorer settings on the Vulnerabilities Findings page, bookmark the full page URL. Your search query and facet selections are preserved in the URL.</div>
```
